recipes/wasmtime/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class WasmtimeTestConan(ConanFile):
+    settings = 'os', 'compiler', 'build_type', 'arch'
+    generators = 'cmake', 'cmake_find_package'
```

Comment:
Don't use `cmake_find_package`.
Because upstream wasmtime does not provide cmake targets, we don't test/use it in a test package.

Reply:
It breaks build

`wasmtime::wasmtime` helps in way to add all includes and necessary flags

I do not see any issue with verifing using `cmake_find_package` because I verify linking with test_package

Reply:
What breaks? It should not.
If it breaks somewhere, that means something else is wrong.

Reply:
@madebr If something else would be wrong that Ci did not pass ...

The difference between `wasmtime::wasmtime` and `${CONAN_LIBS}` is that `wasmtime::wasmtime` implicitly include: include libraries, defines and etc.

Reply:
> @madebr If something else would be wrong that Ci did not pass ...

I don't understand this.

> The difference between `wasmtime::wasmtime` and `${CONAN_LIBS}` is that `wasmtime::wasmtime` implicitly include: include libraries, defines and etc.
> 
> It is just much easier to write with `wasmtime::wasmtime` than with `${CONAN_LIBS}`

Easy is relative.
Right now you're mixing global includes and a cmake target, which is even more potentially troubling.

I'm referring to this section in the docs: https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#test-package
Because wasmtime does not export a wasmtime cmake_find_package generator name, it's not tested.